src/main/scala/firrtl/package.scala

```diff
@@ -21,4 +21,21 @@ package object firrtl {
   type WSubIndex = ir.SubIndex
   type WSubAccess = ir.SubAccess
   type WDefInstance = ir.DefInstance
+
+  @deprecated("Use firrtl.backends.verilog.VerilogEmitter", "1.4")
```

Comment:
This comment didn't get into the review...

This doesn't align with the usage of `@deprecated` in this file, but it's a good idea to include the name of the package in the thing you're deprecating. The following aligns with Scala's guidelines on use of `@deprecated`:

```suggestion
  @deprecated("Use firrtl.backends.verilog.VerilogEmitter", "FIRRTL 1.4")
```